docs/src/administration/pricing/_index.md

```diff
@@ -0,0 +1,138 @@
+2. Click **Edit plan**.
+3. Under **Environments application size**, select a new size.
+4. Click **Save** to accept the new plan cost.
+
+This increases the size for all app containers.
+It doesn't affect service containers.
+
+## Switch plans
+
+To switch plans, follow these steps:
+
+1. In your project in the Console, click **{{< icon settings >}} Settings**.
+2. Click **Edit plan**.
+3. Under **Plans**, select a new plan.
+4. Click **Save** to accept the new plan cost.
+
+## Extras
+
+In addition to the included resources for a plan, you can add extras to help scale your project.
+
+All extra consumption is prorated to the time it was actually used.
+For example, if you add an extra user for 10 days,
+you're billed around 1/3 of the current price for a full month.
+
+You can add the following extras:
+
+* Users -- for an additional fee per user, except at certain [agency levels](#agencies).
+* Environments -- extra staging/development environments can be added in multiples of 3.
+* Storage -- extra storage can be added for a fee per 5&nbsp;GB per environment.
+  So if you have added 3 additional environments for a total of 6 development environments,
+  the fee is multiplied by 6.
+  
+  You can make any changes up to 100&nbsp;GB yourself.
+  For storage more than 100&nbsp;GB, create a [support ticket](https://console.platform.sh/-/users/~/tickets).
+
+  You currently can't reduce your storage on your own for technical reasons.
+  If you need to reduce storage, create a [support ticket](https://console.platform.sh/-/users/~/tickets).
+
+## Multiple apps in a single project
+
+All plans support multiple containers, but the containers share the plan's global resources.
+
+The resources of a Standard plan can support a single app with services.
+But they **aren't sufficient to run more than one app** with a service.
+Useful multi-app projects start at {{< partial "plans/multiapp-plan-name" >}}.
+For example, a {{< partial "plans/multiapp-plan-name" >}} plan can support 3 apps
+with a MySQL service and a Redis service.
+
+If you aren't sure whether a specific setup would fit into a plan,
+don't hesitate to contact support.
+
+## {{% names/dedicated-gen-3 %}}
+
+For more resources along with triple redundancy on every element of the stack, use a {{% names/dedicated-gen-3 %}} plan.
+
+Learn more about [{{% names/dedicated-gen-3 %}}](../../dedicated-gen-3/overview.md).
+
+To discuss how {{% names/dedicated-gen-3 %}} could work for you, [contact Sales](https://platform.sh/contact/).
+
+## Agencies
+
+Agencies have access to several partner levels with many perks.
+
+* Agency partner levels: Registered, Bronze, Silver, Gold, Platinum, and Diamond
+* Free user licenses: Silver and above
+* Free development plan: Gold and above
+* Access to an agency-speciﬁc Small plan price: Bronze and above
+* Agency discounts for plans depending on the combination of the account tier and agency partner level:
```

Comment:
:question:
What's the `account tier` in that context?
Is this [the `Partner tier`](https://platform.sh/solutions/digital-agencies/) that changes with the credits?

:pick:
If there are no items for registered and Bronze, why not add `-` in the cells?

Reply:
The account tier is listed in the first column of the table.